docs/guide/TNGF/tngfue-installation.md

```diff
@@ -57,17 +63,33 @@ cd tngfue
 
 ### TNGFUE Configuration
 
-#### Build/Compile Configuration
+* Use the `prepare` script:
 
-Go to `wpa_supplicant/`
+(Optional) Open the script and adjust it with the desired parameters/vars
 ```
-cd wpa_supplicant
+nano prepare.sh
```

Comment:
also edit run.sh `IFACE_NAME`

Reply:
To make it clear, I should let the user dynamically change the Wi-Fi interface when calling the run.sh script, right? Because, from my point of view, the interface name should be present/updated on sec.conf too. Perhaps we could instead update the message on line 73 to make clear that the names must match on both sh scripts (run and prepare), what do you think?

Reply:
how about reading the `wifiiname` variable directly from sec.conf and using it as a parameter for wpa_supplicant?

Reply:
Ok. It is done in https://github.com/free5gc/tngfue/pull/2/commits/e0c498d8ac969b63dd81ec1ffe59e760a022dcca